### PR TITLE
Feat/asr alignment

### PR DESCRIPTION
**Pull request by JoelSaleem**

### CODE REVIEW

File: dags/pipeline.py [hunk 1]
```diff
@@ -120,7 +120,9 @@ def get_pod_operator(command, task_id, do_xcom_push=False):
 )
 
 prepare_wavs_for_downsampling = get_pod_operator(
-    command="python /usr/tasks/prepare_wavs_for_downsampling.py {{ dag_run.conf['skip_list'] }} '{{ task_instance.xcom_pull('check_16_bit')['git_hash'] }}'",
+    command="python /usr/tasks/prepare_wavs_for_downsampling.py" +
+    " {{ dag_run.conf['skip_list'] }}" +
+    " '{{ task_instance.xcom_pull('check_16_bit')['git_hash'] }}'",
```
Comment (JoelSaleem) [inline dags/pipeline.py:125]: this is just formatting, no changes here

File: tasks/check_16_bit.py [hunk 2]
```diff
@@ -24,13 +25,13 @@ def run():
 
     makedirs(f'/usr/foo/wav_16bit_{git_hash}')
 
-    for wav in listdir('/usr/foo/wavs'):
+    for wav in listdir('/usr/foo/wav_raw'):
```
Comment (JoelSaleem) [inline tasks/check_16_bit.py:28]: wrong dir name. run.sh looks for the wav_raw dir

**Review by udoniyor — APPROVED** [review]
I would suggest to extract out different directory names into a constants i.e.:

`/usr/foo` and many others...

**Comment by JoelSaleem** [comment]
> I would suggest to extract out different directory names into a constants i.e.:
> 
> `/usr/foo` and many others...

Yes will do, but probably after the chain is complete since I have a few prs on the go at the moment and it will cause many conflicts. 
The plan is to have a constants file with: 
MOUNTED_DIR = '/usr/foo' (which actually will be renamed to /usr/data)
WAV_RAW = f'{MOUNTED_DIR}/{dag_run_id_or_some_identifier}/wav_raw'

**Pull request merged**